src/patternfly/assets/pficon/pficon.scss

```diff
@@ -21,376 +25,376 @@
 }
 
 @function pf-icon-char($filename) {
-	$char: "";
+	$char: '';
 
 	@if $filename == add-circle-o {
-		$char: "\E001";
+		$char: '\E61B';
```

Comment:
I'm not sure if anyone is relying on the existing chars. I'm comfortable making this change as long as you'll be able to fix any potential problems consumers have in the weeks after this is merged.